sever_test.go

```diff
@@ -31,7 +31,63 @@ func TestServer_HTTPHandler_GET(t *testing.T) {
 
 	expectedEvt := RequestEvent{
 		Header:     req.Header,
-		RemoteAddr: "192.0.2.1:1234",
+		RemoteAddr: "192.0.2.1",
+		Method:     "GET",
+		URL:        req.URL,
+		Body:       []byte{},
+	}
+	assert.Equal(t, expectedEvt, events[0])
+}
+
+func TestServer_TrustedHeader(t *testing.T) {
+	brain := joetest.NewBrain(t)
+	conf := config{
+		listenAddr:    "127.0.0.1:0",
+		logger:        zaptest.NewLogger(t),
+		trustedHeader: "x-real-ip",
+	}
+	s := newServer(conf, brain)
+
+	resp := httptest.NewRecorder()
+	req := httptest.NewRequest("GET", "/test", nil)
+	req.Header.Set("x-real-ip", "300.300.300.300")
+	s.HTTPHandler(resp, req)
+
+	brain.Finish()
+	events := brain.RecordedEvents()
+	assert.NotEmpty(t, events)
+
+	expectedEvt := RequestEvent{
+		Header:     req.Header,
+		RemoteAddr: "300.300.300.300",
+		Method:     "GET",
+		URL:        req.URL,
+		Body:       []byte{},
+	}
+	assert.Equal(t, expectedEvt, events[0])
+}
+
+func TestServer_TrustedHeaderMultiple(t *testing.T) {
```

Comment:
You would think a bot would have more attention to detail to notice that Lune 81 differs.

Reply:
I would ok with disabling this linter:

```diff
diff --git a/.golangci.yml b/.golangci.yml
index 09cd00d..811e18a 100644
--- a/.golangci.yml
+++ b/.golangci.yml
@@ -36,6 +36,7 @@ linters:
     - prealloc
     - depguard
     - interfacer
+    - dupl
 
 service:
   golangci-lint-version: 1.15.x # use the fixed version to not introduce new linters unexpectedly
```

WDYT?

Reply:
Alternatively, could the threshold be tweaked?
```
  dupl:
    threshold: 100
```

Reply:
Good idea :+1:

Reply:
So I pulled up the documentation for the tool, https://github.com/mibk/dupl. Since it only operates on types, it has no way of differentiating the values. The threshold sets the number of AST tokens that have to match. So we could change it to only match longer duplicated sections or, like you originally suggested, remove it entirely. Seeing how it works, I'm now partial to removing it.